packages/core/src/server/utils/initial-state.ts

```diff
@@ -2,13 +2,41 @@ import deepmerge from "deepmerge";
 import { NormalizedSettings } from "@frontity/file-settings";
 import { Package } from "@frontity/types";
 
-export default ({
-  settings,
-  url,
-}: {
+/**
+ * Options for {@link state}.
+ */
+interface StateOptions {
+  /**
+   * Settings for a frontity project.
+   * They are defined in a [frontity.settings.js](https://docs.frontity.org/learning-frontity/project#the-frontity-setting-js-file) file.
+   */
   settings: NormalizedSettings;
+  /**
+   * The WHATWG parsed URL object for the current page. Passed from the
+   * [context](https://koajs.com/#context) of the Koa app.
+   */
   url: URL;
-}) => {
+}
+
+/**
+ * Initialize the state of a Frontity project.
+ *
+ * @param options - Defined in {@link StateOptions}.
+ * @returns The Frontity state object.
+ */
+const state = ({ settings, url }: StateOptions) => {
+  const paramsToDelete = ["frontity_source_auth", "frontity_name"];
```

Comment:
I don't think that's a good idea because these options are meant to be used also by packages, like `frontity_source_auth`, and the blacklist is hardcoded in the core. So each time a package creator wants to use one of these options in their package, they'd have to do a PR to the core to add it.

Do you see any benefit on maintaining this blacklist?